packages/compass-web/src/entrypoint.spec.tsx

```diff
@@ -82,6 +85,7 @@ describe('CompassWeb', function () {
         {...props}
         // @ts-expect-error see component props description
         __TEST_MONGODB_DATA_SERVICE_CONNECT_FN={connectFn}
+        __TEST_CONNECTION_STORAGE={storage}
```

Comment:
Do we need to pass a test one now that it's the same thing that's being set up? (and it doesn't look like we're running any assertions against it)